sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -255,7 +255,11 @@ private static String produceMeasurement(
     schemaResults.put("timestamp", timestamp);
     schemaResults.put("runner", runner);
     schemaResults.put("measurement", measurement);
-    schemaResults.put("runtimeMs", runtimeMs);
+    schemaResults.put(
+        "runtimeMs",
```

Comment:
We should do the same for `numResults`. Now the schema that is being created is:
```
fieldKey   fieldType
--------   ---------
numResults float
runtimeMs  integer
```
And we need `numResults` as an integer too. I'm sorry I didn't notice that earlier.